scripts/run_backend_tests.py

```diff
@@ -343,15 +341,10 @@ def main(args=None):
     dev_appserver.fix_sys_path()
 
     if parsed_args.generate_coverage_report:
-        python_utils.PRINT(
-            'Checking whether coverage is installed in %s'
-            % common.OPPIA_TOOLS_DIR)
-        if not os.path.exists(
-                os.path.join(common.OPPIA_TOOLS_DIR, 'coverage-4.5.4')):
-            python_utils.PRINT('Installing coverage')
-            install_third_party_libs.pip_install(
-                'coverage', '4.5.4',
-                os.path.join(common.OPPIA_TOOLS_DIR, 'coverage-4.5.4'))
+        python_utils.PRINT('Checking whether coverage is installed')
+        if subprocess.call(['which', 'coverage']) != 0:
```

Comment:
We generally always use local versions. Why can't we do that in this case? Using local versions makes it easier for Oppia to run properly on different machines.

Also, we should add coverage to our standard list of dependencies so that it's installed the same way as the other Python packages that we depend on.

Reply:
The interesting thing is that once I use the local version, the following error occurs: 
```
Traceback (most recent call last):
  File "oppia_tools/coverage-4.5.4/bin/coverage", line 6, in <module>
    from coverage.cmdline import main
ImportError: No module named coverage.cmdline
```

I've attempted to set the path and that error still exists (with the exception that I can autocomplete the `coverage` command).

Reply:
This issue was "hidden" in CircleCI since it previously installed it at the system level and the "local" level. The module was fetched from the system version, so we didn't that issue before.

Reply:
Looks like there was an attempt previously https://github.com/oppia/oppia/pull/7465/files#r320273705

Reply:
Odd -- not sure what's happening here, but I don't understand why it wouldn't work. Does searching for that error message help? I'm occupied most of today, but would be happy to try and poke at it a bit later in the evening if you haven't found a way to get it working yet!

Reply:
So the solution to this was to add it to the `PYTHONPATH`. That being said, I would like someone to assist in testing this to make sure I don't break anything else.